### PR TITLE
[plugin.video.mediathek] 0.7.4

### PR DESCRIPTION
**Pull request by raptor2101**
CHG: Repearing NDR und 3SAT

second try ...

### CODE REVIEW

File: plugin.video.mediathek/addon.xml [hunk 1]
```diff
@@ -2,10 +2,10 @@
 <addon
   id="plugin.video.mediathek"
   name="Mediathek"
-  version="0.7.3"
+  version="0.7.4"
   provider-name="Raptor 2101">
   <requires>
-   <import addon="xbmc.python" version="2.24.0"/>
+   <import addon="xbmc.python" version="2.1.0"/>
```
Comment (learningit) [inline plugin.video.mediathek/addon.xml]: xbmc.python version should be 2.14.0 for Gotham. The rest looks ok.

**Comment by raptor2101** [comment]
Two questions:

- Should a addon import a dependency at a minimal level needed or required by the kodi version? In 0.7.3 I accidentally raised the version 2.1.0 to 2.24.0. There is no need within the Plugin to rase the API level. will this version be dropped in further kodi versions?

- to raise the level to 2.14.0 is there a new PR needed or can I simply commit a change to the addon.xml

**Comment by learningit** [comment]
You just have to make the change, commit it and then squash the commits. See https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

No need for a new PR

**Comment by raptor2101** [comment]
done ...

**Comment by learningit** [comment]
Just one thing - you need to rename the commit to the correct format, i.e. [plugin.video.mediathek] 0.7.4

**Comment by learningit** [comment]
Actually it doesn't look like the squash was done correctly, you should end up with a single commit.
Please make sure you name it correctly when you edit the pick/squash file.

**Comment by raptor2101** [comment]
... postet to early ...

**Comment by learningit** [comment]
Sorry all looks fine now, I didn't refresh the browser.

**Pull request merged**